addons/website/static/src/scss/website.scss

```diff
@@ -435,6 +435,12 @@ font[class*='bg-'] {
     padding: 2px 6px 4px;
 }
 
+.lead {
+    // Use the multiplier to adapt according to user's preference.
+    line-height: o-website-value('body-line-height') * $o-lead-ratio;
+    margin-bottom: o-website-value('paragraph-margin-bottom') * $o-lead-ratio;
+}
```

Comment:
`line-height` is not necessary indeed. My bad, I've "misread" the UI and assumed that the paragraph's line-height controller was affecting `<p>` tags, while it's affecting the main `line-height` value indeed 👍 
![image](https://github.com/odoo/odoo/assets/8558351/847f6923-7a63-4143-9b35-951589f1cf2e)

`margin-bottom`  is needed instead, since the bottom gap should be proportional to the paragraph's one. I'd suggest to keep the rule here and not in `bootstrap_review` since the aim is to reflect editor's customization rather than changing bootstrap default behaviour.

Reply:
> line-height is not necessary indeed

:+1: 

> not in bootstrap_review since the aim is to reflect editor's customization rather than changing bootstrap default behaviour.

But paragraph-margin-bottom is a bootstrap feature. This line will in any case become: `margin-bottom: $paragraph-margin-bottom * $o-lead-ratio;` (in general, I prefer not use `o-website-value` when it is linked to a bootstrap one). So should not it be a bootstrap extension if you judge it necessary?

In fact what about margin-top? That one should indeed not be in bootstrap_review as it is a website-only feature, but should not it be impacted the same way as the margin-bottom?

Reply:
Removing the whole extra rule for now, you can still re-open a PR after the merge with the change if needed :+1: